server/routes/auth.go

```diff
@@ -87,6 +87,7 @@ func SetAuthRoutes(e *echo.Echo, cfgProvider *config.ConfigProviderWithRefresh)
 
 	api.GET("/sso", authenticate(&config, providerCfg.Options))
 	api.GET("/sso/callback", authenticateCb(ctx, &config, provider))
+	api.GET("/sso_callback", authenticateCb(ctx, &config, provider)) // compatibility with UI v1
```

Comment:
or just make this the only endpoint?

Reply:
Just make it the only one. No need to complicate things.